test/e2e/pre_provisioning_test.go

```diff
@@ -251,7 +252,7 @@ var _ = ginkgo.Describe("Pre-Provisioned", func() {
 				ginkgo.Skip("test case is only available for in-tree driver")
 			}
 
-			skipManuallyDeletingVolume = true
+			skipVolumeDeletion = true
```

Comment:
Do we need to skip the deletion here?

Reply:
I assumed that this was intended by the original author. Is there any reason not to delete the created volume here?

Reply:
An inline volume should automatically be deleted by K8s/CSI when the pod is deleted. There shouldn't be a volume delete after the test runs.

Reply:
Sorry for unresolving this conversation, it seems like this test is a duplicate? We have two tests that create and tests this inline volume scenario. Is that intentional?

Reply:
https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/4b4960217399b8d834926cf7a824f7a9bb2447bf/test/e2e/pre_provisioning_test.go#L249
https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/4b4960217399b8d834926cf7a824f7a9bb2447bf/test/e2e/pre_provisioning_test.go#L249

Reply:
It appears the first was added to handle testing with different schedulers. Since this test is targeting the in-tree driver, do we even need to test with the scheduler extender? It seems like we should remove this test case.

Reply:
That sounds odd :0 And taking a quick look, the second test seems to be reserved for migration test while the first is set to be run for both.

Reply:
And currently `pre_provisioned_inline_volume_tester` is set to take scheduler name as a param. I wonder if this should revised as well?

https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/4b4960217399b8d834926cf7a824f7a9bb2447bf/test/e2e/testsuites/pre_provisioned_inline_volume_tester.go#L37